lib/etna.rb

```diff
@@ -6,4 +6,6 @@
 
 ROOT_PATH = File.dirname(__FILE__)
 Dir.glob(ROOT_PATH + '/etna/concerns/*.rb') { |file| require file }
+Dir.glob(ROOT_PATH + '/etna/components/response.rb') { |file| require file }
+Dir.glob(ROOT_PATH + '/etna/errors/base.rb') { |file| require file }
 Dir.glob(ROOT_PATH + '/etna/**/*.rb') { |file| require file }
```

Comment:
So this `**/*.rb` doesn't work for mass requiring? 
If not, please remove. If yes, lines 8 - 10 are not necessary right?